webdevbroz/pomodoro-app
# Add page layout

## Goal
This will be the layer for the overall application. Responsiveness included.

Related issue #https://github.com/webdevbroz/pomodoro-app/issues/9

Things to note:
- custom utility class names have been added to `tailwind.config.ts` for specific heights and widths, according to the Figma design.
- white boarder boxes have been added for now to represent the top layer structure and placement of component. This makes it easier to visualize the implementation. Be sure to remove `border-4 border-white` on the container `div` after adding a component.

## How to test

- Go between desktop, tablet, and mobile to check responsiveness.
- run `yarn test` to make sure the test for the Logo component passes.

## Screenshots
![image](https://github.com/webdevbroz/pomodoro-app/assets/27636896/4e520107-0cee-44ba-bc72-2abaf6d09482)
